_commands/tools/time.md

```diff
@@ -0,0 +1,119 @@
+---
+---
+
+time
+---
+`time` prints out the total time elapsed, the time used by system overhead and the time used to execute the utility.
+	
+<!--more-->
+
+### Situation
+
+If you simply want to output the time it takes to run a command
+	
+~~~ bash
+$ time ./project -m < input.txt
+~~~
+
+### Meaning:
+`time` calls the bash time command, `./project`is the program you are trying to run,
+`-m` is a flag for that program and `< input.txt `is the input file for the program.
+Anything that appears after `./project` is taken like it would be without using the time command.
+This will output the following:
+
+~~~ bash
+0.00user 0.00system 0:00.00elapsed 
```

Comment:
I ran it one the virtual machine, when I ran it on my Mac it only output user system and real time.

Steven Lengieza
University of Michigan 2018
College of Engineering
Michigan Men's Club Lacrosse-Apparel Chair

> On Mar 25, 2016, at 11:28 PM, Matt Terwilliger <notifications@github.com> wrote:
> 
> In _commands/tools/time.md:
> 
> > +### Situation
> > +
> > +If you simply want to output the time it takes to run a command
> > +	
> > +~~~ bash
> > +$ time ./project -m < input.txt
> > +~~~
> > +
> > +### Meaning:
> > +`time` calls the bash time command, `./project`is the program you are trying to run,
> > +`-m` is a flag for that program and `< input.txt `is the input file for the program.
> > +Anything that appears after `./project` is taken like it would be without using the time command.
> > +This will output the following:
> > +
> > +~~~ bash
> > +0.00user 0.00system 0:00.00elapsed 
> My time just outputs real/user/sys, not the stuff below. What OS are you on?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>